pkg/storage/consistency_queue.go

```diff
@@ -104,6 +104,11 @@ func (q *consistencyQueue) process(
 	}
 	req := roachpb.CheckConsistencyRequest{}
 	if _, pErr := repl.CheckConsistency(ctx, req); pErr != nil {
+		// Check again here to see if it is turned off just in case there was a race
+		// when starting up the queue.
+		if q.interval() <= 0 {
```

Comment:
This seems kind of suspicious - it doesn't really fix the race, right? Since repl.CheckConsistency still gets called.

Reply:
I'm not really sure how much this matters, but I'd defer to a core person for this part of the review, I guess.

Reply:
Yeah, I'm pretty sure, every now and then the queue gets initialized before the cluster setting is set.  I can remove this fix and it still quiets most of the errors.

Reply:
Ok, after a discussion with @nvanbenschoten , we looked for the source of the problem.  And it turns out it was an error in conditional logic on the if statement below.  Should be fixed now.